### DISCUSSION TITLE
Detect subscript and superscript with pymupdf?

### DISCUSSION THREAD

**Discussion by CaiSamuelsFSA** [pymupdf/PyMuPDF#3286] [Looking for help] [answered]
I currently have a script that extracts text from a PDF file and converts it into HTML.

Some of the documents include superscript for references and footnotes.

Is there a way for pymupdf to detect subscript and superscript so I can output the HTML equivalent?

**Comment by JorjMcKie** [comment, marked as answer]
You can detect detect (many, not all) superscripts by checking the `span["flags"]` in `page.get_text("dict", ...)`, same with "rawdict".
There is no way to detect subscripts.
